static/src/stylesheets/module/content-garnett/_pillars.scss

```diff
@@ -1,8 +1,7 @@
-@mixin colorPalatte($pillarColor, $kickerColor, $mediaColor) {
+@mixin colorPalatte($kickerColor, $pillarColor, $mediaColor) {
```

Comment:
That's a bold move, but I suppose you've tested it 😂

Reply:
Ha, maybe tested a little quickly, will double check it wasn't 6PM madness.